src/test/java/org/sasanlabs/service/vulnerability/commandInjection/CommandInjectionTest.java

```diff
@@ -0,0 +1,592 @@
+package org.sasanlabs.service.vulnerability.commandInjection;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.doReturn;
+import static org.mockito.Mockito.verify;
+import static org.sasanlabs.vulnerability.utils.Constants.LOCALHOST;
+
+import java.io.IOException;
+import java.net.URI;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.logging.log4j.util.Strings;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+import org.sasanlabs.service.exception.ServiceApplicationException;
+import org.sasanlabs.service.vulnerability.bean.GenericVulnerabilityResponseBean;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.RequestEntity;
+import org.springframework.http.ResponseEntity;
+
+/*
+ * @author Feng-Yuan Yang coolesaltw@hotmail.com
+ */
+public class CommandInjectionTest {
+
+    private CommandInjection commandInjection;
+    private static final String ipAddress = "10.29.28.27";
```

Comment:
I think the simplest way is to have 127.0.0.1 which will work for localhost as well as level 6.
can you please update https://github.com/SasanLabs/VulnerableApp/pull/221/files#diff-d7958c1340593be6c9fb8b7ec2d7935aR11 with 127.0.0.1 address.